_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Optimising UX for Software"
+date: 2021-04-27
+tags: ux devops design
+author: Lucy Sharp
+image: /img/User_Journey_Map.jpeg
+---
+
+My baptism into the cult of Mechanical Rock kicked off with a sword fight (legally, I can't say more) and it got me thinking about the relationship between UX and DevOps. Particularly, how DevOps drives the evolution of UX design.
+
+Sword fight aside, there is opportunity to accelerate technology delivery through the reciprocal relationship between UX and DevOps especially as UX provides a competitive advantage to software delivery.
+<br /><br />
+
+## What is UX?
+
+User Experience (UX) “encompasses all aspects of the end user's interaction with the company, its services, and its products” ([Human Centred Design in QLD Toolkit](https://www.forgov.qld.gov.au/human-centred-design-resources)).
+
+UX provides insight into the gaps, challenges and opportunities of a problem space within a specific domain to improve user experience and deliver a specific ROI to key stakeholders and customers. For every one dollar invested in UX yields a hundred dollar return; a return of 9,900% as reported by [Forrester](https://www.forrester.com/report/The+Six+Steps+For+Justifying+Better+UX/-/E-RES117708).
+
+Unfortunately, the inherent value of UX is often not recognised and this may reflect designers ways of working. Traditionally, design operates in a more waterfall environment where siloed work coupled with a lack of understanding about how the system operates impacts user experience and leads to design re-work or developers making decisions without a UX designer ([Talia Lavie](https://techbeacon.com/devops/8-ways-integrate-ux-speed-devops)). Such siloed work between designers and developers is a constraint; slowing down development and delivery. With the evolution of processes and culture, UX practices need to change. A driver of this evolution is DevOps.
+<br /><br />
+
+## DevOps
+
+For some, DevOps is a buzzword. They trott it out and throw it around in conversation. For others, DevOps is the How. It’s a practice. A way of working. Or the “foreground to a great cult,” to quote Hamish.
+
+So what is DevOps? When I spoke to Nick about this, he could not help himself but take me back to Toyota and manufacturing. Despite my protests of, “I know this,” Nick persisted and I’m glad he did. What I learnt from our conversation was twofold.
```

Comment:
You refer to Nick and Hamish, but the reader probably has no idea who these monkeys are

Reply:
Yeh that is true! I could link to your LinkedIn accounts or add your surnames?